src/Quizzer.DAL.Tests/RepositoryTests.cs

```diff
@@ -0,0 +1,31 @@
+using Quizzer.API.DAL;
+using Quizzer.API.DAL.UnitOfWork;
+
+namespace Quizzer.DAL.Tests {
+    public class RepositoryTests {
+
+        protected UnitOfWorkFactory UnitOfWorkFactory;
+        protected IUnitOfWork UnitOfWork;
+        protected QuizzerDbContext DbContextSUT { get; }
+
+        public async Task DisposeAsync() {
+            await DbContextSUT.Database.EnsureDeletedAsync();
+            await DbContextSUT.DisposeAsync();
+            _ = UnitOfWork.DisposeAsync();
+        }
+
+        public async Task InitializeAsync() {
+            await DbContextSUT.Database.EnsureDeletedAsync();
+            await DbContextSUT.Database.EnsureCreatedAsync();
+            UnitOfWork = UnitOfWorkFactory.Create();
+        }
+
+        [Fact]
+        public void Test1() {
+
+            var repo = UnitOfWork.GetRepository<QuizEntity>();
+
+            //Assert.
+        }
```

Comment:
This should not be here, also what is then the purpose of this file?

Reply:
It was duplicit project, removed.